Rubberduck.Parsing/Inspections/Resources/InspectionsUI.de.resx

```diff
@@ -784,49 +784,49 @@ Falls der Parameter 'null' sein kann, bitte dieses Auftreten ignorieren. 'null'
     <value>Ein leerer 'Case' Block ohne ausführbare Anweisungen lässt andere Entwickler über die Absicht des Codes im unklaren. Vermeiden Sie das Schreiben von unnötigem Code.</value>
   </data>
   <data name="StopKeywordInspectionResultFormat">
-    <value>Stop keyword detected</value>
+    <value>Das 'Stop' Schlüsselwort unterbricht die Ausführung</value>
   </data>
   <data name="EmptyForEachBlockInspectionMeta">
-    <value>A Loop block should have executable statements</value>
+    <value>Eine leere 'For Each...Next' Schleife ohne ausführbare Anweisungen lässt andere Entwickler über die Absicht des Codes im unklaren. Vermeiden Sie das Schreiben von unnötigem Code.</value>
   </data>
   <data name="EmptyElseBlockInspectionMeta">
-    <value>An Else block should have executable statements</value>
+    <value>Ein leerer 'Else' Block ohne ausführbare Anweisungen lässt andere Entwickler über die Absicht des Codes im unklaren. Vermeiden Sie das Schreiben von unnötigem Code.</value>
   </data>
   <data name="RemoveEmptyElseBlockQuickFix">
-    <value>Remove empty Else block</value>
+    <value>Entferne leeren 'Else' Block</value>
```

Comment:
Again consistency in form: _Entfernen Sie den_...

Reply:
Auch hier: keine Nutzerhandlung

Reply:
Ah, das macht Sinn. Ich hatte den Kontext irgendwie falsch im Kopf.